### ISSUE TITLE
How to calculate the weights of the CrossEntropyLoss, i.e. data/s2d3d_sem/label13_weight.pth?

### ISSUE THREAD

**Issue by zcq15** [sunset1995/HoHoNet#2]
The codes use the weights of 13 classes to train the segmentation networks, but how to calculate the weights of different classes for the cross-entropy loss? There is no corresponding part in the pre-processing codes or uploaded file in the repository. Could you please provide the values of the weights? Thanks!

**Comment by Zishuo-9** [comment]
I have the same problem, have you solved it？

**Comment by sunset1995** [comment]
We use the `label_weight` in [this](https://github.com/meder411/Tangent-Images/blob/master/experiments/semantic_segmentation/ss/models/build.py#L105), which is:
```
tensor([28.4696, 21.7185, 11.3940,  8.8658, 17.5377,  7.7713, 20.2949,  8.1953,
        10.0595, 43.4360, 20.9806,  3.2783, 20.9854])
```

I have added it to `assets/label13_weight.pth`.
Thanks for reminding me :)
